src/components/pages/Dashboard.jsx

```diff
@@ -0,0 +1,31 @@
+import { logout } from "../../util/fetchData";
+import { useAuth } from "../../AuthProvider";
+import { useNavigate } from "react-router-dom";
+
+const Dashboard = () => {
+    // temporarily this functionality is here, then it will be moved to the header
+    
+    const { logoutAction, userData } = useAuth();
+    const navigate = useNavigate();
+
+    const handleLogout = async () => {
+        logoutAction();
+        logout()
+            .then(() => {
+                logoutAction();
+                navigate('/');
+            })
+            .catch(error => {
```

Comment:
You are directly working with promises here while using axios in another file. Either is fine. Let's just be consistent with which approach we use.